src/index.ts

```diff
@@ -4,6 +4,7 @@ import ResultsCalculator from './services/results-calculator';
 
```

Comment:
Is it possible to import only the methods you need from your dependencies?

For instance, could you say:
```
import { createInterface } from 'readline';
import { createReadStream } from 'fs';
```